site.yml

```diff
@@ -21,6 +21,34 @@
       delegate_to: localhost
       changed_when: false
 
+    - name: Create folder to store config
+      file: path={{ jenkins_job_config_file_src }} state=directory mode=0755 owner=jenkins group=jenkins
+
+    - name: Locally clone Jenkins job configs
+      git:
+        repo: "{{ jenkins_job_config_git_src }}"
+        dest: "/tmp/nfv_jobs_config"
```

Comment:
Can you make this clone relative to `./files/nfv_jobs_config` and then add the `nfv_jobs_config` to the `.gitignore` file? I did the same thing for another clone. I found sync'ing from the /tmp dir wasn't as consistent for some reason.